### DISCUSSION TITLE
I have upload student id and unofficial transcript, but can’t get verified

### DISCUSSION THREAD

**Discussion by guocity** [community/community#49743] [GitHub Education]
I have upload student id and unofficial transcript, but can’t get verified

**Comment by jge162** [comment]
Haver you also reached out to support? Perhaps they can help further. Here is [link](https://support.github.com/contact/education).

**Reply by guocity** [reply]
Thanks for the link, it works

**Reply by samus-aran** [reply]
Appreciate the quick response @jge162 but we had recently [announced changes to GitHub Education support ](https://github.com/orgs/community/discussions/49366)informing users that we don't provide individual reviews of applications anymore. We encourage users to provide more information about their rejection reasons in the community if they are unable to find similarly asked questions. 👍

**Reply by jge162** [reply]
I understand. Thanks for update.

**Comment by samus-aran** [comment]
Hi @guocity 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
